src/dfx/src/commands/identity/export.rs

```diff
@@ -14,7 +15,10 @@ pub struct ExportOpts {
 pub fn exec(env: &dyn Environment, opts: ExportOpts) -> DfxResult {
     let name = opts.identity.as_str();
 
-    let pem = IdentityManager::new(env)?.export(name)?;
+    let pem = IdentityManager::new(env)
+        .context("Failed to load identity manager.")?
+        .export(name)
+        .with_context(|| format!("Failed to export {}.", name))?;
```

Comment:
Another example where we are adding, from a higher level, context that could be added at a lower level.